GA-worksheet.md

```diff
@@ -26,44 +31,58 @@
 ##Project #2 Part #2: Style the Header with CSS
 
 1. What advantages does the `<link>` tag have over the `<style>` tag?
+    You can its better in an orginization sense.
```

Comment:
It definitely helps with organization. `<style>` lets you apply styling to the HTML document, and `<link>' lets you connect an external resource, like a CSS document.